src/main/java/frc/robot/commands/drivetrain/fieldRelative.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Drivetrain;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class fieldRelative extends InstantCommand 
+{
+  private Drivetrain drivetrain = RobotContainer.drivetrain;
```

Comment:
Don't access the subsystem statically, instead have a parameter that passes it in. Also make sure class members are denoted properly.